DesignSafe-CI/portal
# Date of Publication issue

## Overview: ##
Date of Publication was always showing the newest date, meaning if a version or amend was done, then the Date of Publication was updated. This is an issue because the Date of Publication should always show the actual date if was published. 

## PR Status: ##


## Related Jira tickets: ##

* [DES-2492](https://jira.tacc.utexas.edu/browse/DES-2492)

## Summary of Changes: ##
For Simulation, FR, Experimental, and Hybrid Sim, I added a dictionary called dateCreated that stores a DOI : dateCreated. In the html, the dictionary is cycled through, and when the entity DOI matches the dictionary DOI, then the dateCreated is displayed. 
For type Other, the Date of Publication is not displayed on the Project side, so no changes were made. 

## Testing Steps: ##
For each project type:
1. Find a project that has versions. 
2. In both  'My Projects' and 'Published' confirm that each entity has it's proper Date of Publication. If it's versioned, these dates should not be all the same. 

For project type other:
1. Confirm that changes are needed.

## UI Photos:
<img width="1175" alt="Screen Shot 2023-05-31 at 10 02 20 AM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/5a6bfdd8-0c80-464c-a79e-5682e2a23ad0">

## Notes: ##
